poller.php

```diff
@@ -110,12 +110,27 @@
     $config['noinfluxdb'] = true;
 }
 
+if (isset($options['g'])) {
+    $config['nographite'] = true;
+}
+
 if ($config['noinfluxdb'] !== true && $config['influxdb']['enable'] === true) {
     $influxdb = influxdb_connect();
 } else {
     $influxdb = false;
 }
 
+if ($config['nographite'] !== true && $config['graphite']['enable'] === true) {
+    if (isset($config['graphite']['port'])) {
+        $graphite_port = $config['graphite']['port'];
+    } else {
+        $graphite_port = '2003';
+    }
```

Comment:
Set the default in includes/defaults.inc.php. This entire isset piece is then not needed.